src/topp/FidoAdapter.cpp

```diff
@@ -224,7 +279,12 @@ public TOPPBase
                                             __PRETTY_FUNCTION__, msg);
       }
       
-      graph_out << "e " << hit.getSequence() << endl; // remove modifications?
+      graph_out << "e " << hit.getSequence().toUnmodifiedString() << endl; // remove modifications?
+      // Julianus: I would say yes, better we do it here, than letting Fido
+      // do some Stringmagic maybe merging peptides that are actually not the same.
+      // Does Fido follow the same convention for Mod naming?
+      // I think it just removes everything not an AA. Means mod labels
+      // like Dimethyl become AAs.
```

Comment:
can you resolve the comments into something that is useful for future developers rather than a discussion?

please: 
- explain what the code is doing
- why this choice was made
- potentially which assumptions are behind this